<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIClientTcp</class>
 <widget class="QWidget" name="UIClientTcp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UIClientTcp</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>681</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_Messages">
    <item>
     <widget class="QLabel" name="label_Messages">
      <property name="text">
       <string>Messages</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_Messages"/>
    </item>
    <item>
     <widget class="QLabel" name="label_Etat">
      <property name="text">
       <string>État</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_Etat"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>681</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_Infos">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_AllInfos">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_IP">
        <item>
         <widget class="QLabel" name="label_IP">
          <property name="text">
           <string>IP Serveur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_IP">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_Port">
        <item>
         <widget class="QLabel" name="label_Port">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_Port">
          <property name="minimum">
           <number>1025</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>8888</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_Texte">
        <item>
         <widget class="QLabel" name="label_Texte">
          <property name="text">
           <string>Texte à envoyer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Texte">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_AllPushButtons">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_PushButtons">
        <item>
         <widget class="QPushButton" name="pushButton_Connexion">
          <property name="text">
           <string>Connexion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Envoyer">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Envoyer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Quitter">
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>UIClientTcp</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>641</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
